src/factory-in-proc.js

```diff
@@ -3,44 +3,20 @@
 const defaults = require('lodash.defaultsdeep')
```

Comment:
why duplicate if factory-in-proc share %90+ code with factory-daemon?

Reply:
We should have a base class then. The main reason is that you want to avoid having factory-daemon being loaded in the browser (and all of its dependencies).